github3/repos/branch.py

```diff
@@ -372,6 +395,93 @@ def users(self, number=-1):
         from .. import users
         return self._iter(int(number), self.users_url, users.ShortUser)
 
+    @decorators.requires_auth
+    def add_teams(self, teams):
+        """
```

Comment:
Please match the style of docstrings and amount of content as the rest of the library.

Reply:
This still needs to match the existing style, e.g.,

```py
"""Add teams to the protected branch restrictions.

See also
https://developer.github.com/...

...
"""
```